package.json

```diff
@@ -96,6 +99,7 @@
     "path-browserify": "^1.0.1",
     "prettier": "^2.4.1",
     "prettier-eslint": "15.0.1",
+    "prismjs": "^1.29.0",
```

Comment:
To make bisecting easier in case of issues, could we keep using `1.19.0` in this PR, and upgrade separately?